### ISSUE TITLE
'self signed certificate in certificate chain' during install

### ISSUE THREAD

**Issue by alephx** [TryGhost/Ghost-CLI#931]
I tried to install ghost, but my corporation use ssl proxy.
`ghost install local` ends with:''

Debug Information:                                                                                                            
    OS: Microsoft Windows, v10.0.17134                                                                                        
    Node Version: v10.15.3                                                                                                    
    Ghost-CLI Version: 1.10.0                                                                                                 
    Environment: development                                                                                                  
    Command: 'ghost install local'                                                                                            
An error occurred.                                                                                                            
Message: 'self signed certificate in certificate chain'                                                                       
                                                                                                                              
Stack: RequestError: self signed certificate in certificate chain                                                             
    at ClientRequest.req.once.err (C:\Users\apala\AppData\Roaming\npm\node_modules\ghost-cli\node_modules\got\index.js:182:22)
    at Object.onceWrapper (events.js:277:13)                                                                                  
    at ClientRequest.emit (events.js:189:13)                                                                                  
    at TLSSocket.socketErrorListener (_http_client.js:392:9)                                                                  
    at TLSSocket.emit (events.js:189:13)                                                                                      
    at emitErrorNT (internal/streams/destroy.js:82:8)                                                                         
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)                                                                 
    at process._tickCallback (internal/process/next_tick.js:63:19)                                                            
Code: SELF_SIGNED_CERT_IN_CHAIN                                                                                               
Path: /ghost/-/ghost-2.22.2.tgz   

Same problem had earlier with installing npm, but was able to solve it with `npm set strict-ssl false`. Is there similar solution for ghost-cli like some flag skip ssl validation or add this certifacate  as trusted?

**Comment by alephx** [comment]
Was able to go little further with downloading ghost.zip and using 
`ghost install local --zip ..\Ghost-2.22.2.zip`.

Got:

--------------- stderr ---------------
warning ghost@2.22.2: The engine "cli" appears to be invalid.
error An unexpected error occurred: "https://registry.yarnpkg.com/debug/-/debug-4.1.1.tgz: self signed certificate in certificate chain".

So installed yarn and set skip ssl with `yarn config set strict-ssl false`.

Now, no success with installing ghost, 

Message: Command failed: yarn install --no-emoji --no-progress
warning ghost@2.22.2: The engine "cli" appears to be invalid.
error https://registry.yarnpkg.com/napi-build-utils/-/napi-build-utils-1.0.1.tgz: Extracting tar content of undefined failed, the file appears to be corrupt: "Invalid tar header. Maybe the tar is corrupted or it needs to be gunzipped?"
warning "@tryghost/members-auth-pages > react-stripe-elements@2.0.3" has unmet peer dependency "react@^15.5.4 || ^16.0.0-0".
warning "@tryghost/members-auth-pages > react-stripe-elements@2.0.3" has unmet peer dependency "react-dom@^15.5.4 || ^16.0.0-0".
error An unexpected error occurred: "ENOENT: no such file or directory, open 'C:\\Users\\<user>\\AppData\\Local\\Yarn\\Cache\\v4\\npm-napi-build-utils-1.0.1-1381a0f92c39d66bf19852e7873432fc2123e508\\node_modules\\napi-build-utils\\.yarn-metadata.json'". 

but it caused by registry.yarnpkg.com is being  blocked by corporate proxy.
So probably if not this blocked host it should be doable to install ghost despite of this ssl error.

**Comment by acburdine** [comment]
Closing this as a duplicate of #570 - there's several issues currently with the way the CLI installs Ghost and how that is affected by certain custom npm connection options.
